Assets/Scripts/EOSAchievementManager.cs

```diff
@@ -205,23 +119,27 @@ public void OnConnectLogin(Epic.OnlineServices.Connect.LoginCallbackInfo loginCa
         /// to any player for whom that information has been requested
         /// previously.
         /// </summary>
-        public void Refresh()
+        public async override void Refresh()
```

Comment:
What is the reason for preferring `async void` signatures instead of `async Task` signatures?

Reply:
This is a good catch. `async void` functions cannot be awaited, whereas `async Task` functions can - moreover they can more properly handle exceptions called from the async context. The default should be `async Task` over `async void`. The latter should really only be used when the function is handling an event.

This instance has been corrected.